Makefile

```diff
@@ -50,6 +50,10 @@ clean:
 docs: ## build the docs on the host
 	$(MAKE) -C docs
 
+.PHONY: lint
+lint: ## do lint processing 
+	golangci-lint run --build-tags="$(BUILDTAGS)"
```

Comment:
I would hard-code the buildtags to produce the same output on every developer machine, like here: 
https://github.com/cri-o/cri-o/blob/97c3b029ad69d7347520e6efc932ac27c8659c33/.golangci.yml#L3-L8

Reply:
I'd like to get @nalind's opinion on this.  He set the calls to a variety of shell scripts to get the value for the platform.  I'm not sure if we can hardcode it without messing up a particular variant on linux.

Reply:
That's an argument, so I guess we can also stick to the current approach.